index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Dr.Norman Borlaug Tribute</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    
+    <script  src="js/index.js"></script>
+    <div id="main" > 
+      
+      <div>
+        <h1 id="title">Dr. Norman Borlaug</h1><br>
+        <div id="caption"><i>The man who saved a billion lives</i></div>
+      </div>
+
+      <div id="img-div">
+        <img id="image" src="https://ichef.bbci.co.uk/news/976/cpsprodpb/E180/production/_106282775_gettyimages-515395982.jpg" alt="Dr. Norman Borlaug">
+      </div>
+    
+      <div>
+        <strong id="tribute-info">Here's a time line of Dr. Borlaug's life:</strong>
+      </div>
```

Comment:
- you did a great job by adding inline comments but it should be better to always include an explanation, a "why", for each requested change so that the next time the student will not repeat the error. :grinning: :grinning: :grinning: